content/library/api/widgets/button.md

```diff
@@ -6,3 +6,13 @@ keywords: button
 ---
 
 <Autofunction function="streamlit.button" />
+
+## Featured videos
+
+Check out our video on how to use one of Streamlit's core functions, the button!
+
+<YouTube videoId="JSeQSnGovSE" />
+
+In the next video, we'll take it a step further and learn how to combine a [button](/library/api-reference/widgets/st.button), [checkbox](/library/api-reference/widgets/st.checkbox) and [radio button](/library/api-reference/widgets/st.radio)!
```

Comment:
What does "In the next video" refer to? Another page, work we'll do in the future?

Reply:
Since people will likely drop into an API page as a standalone, I wouldn't necessarily refer to prior/future videos, but not a strongly held belief either

Reply:
My language there is unclear. I'll push a commit to make it clear that I'm referring to the video below that line.

Reply:
Oh, okay. Something as simple as "In the video below" is probably enough to clarify

Reply:
Done!